Lib/python/pyrun.swg

```diff
@@ -394,18 +409,23 @@ SwigPyObject_repr(SwigPyObject *v)
 {
   const char *name = SWIG_TypePrettyName(v->ty);
   PyObject *repr = SWIG_Python_str_FromFormat("<Swig Object of type '%s' at %p>", (name ? name : "unknown"), (void *)v);
-  if (v->next) {
+  if (repr && v->next) {
     PyObject *nrep = SwigPyObject_repr((SwigPyObject *)v->next);
+    if (nrep) {
```

Comment:
Next, handle OOM on recursively building the representation for the next SWIG objects, to avoid calling `PyUnicode_Concat` or `PyString_ConcatAndDel` below with `nrep = NULL`.